core/src/main/java/io/confluent/kafka/schemaregistry/storage/LookupCache.java

```diff
@@ -106,13 +106,17 @@ public interface LookupCache<K,V> extends Store<K,V> {
    * @param subject the subject
    * @param returnTopLevelIfNotFound whether to return the top level scope if not found
    * @param defaultForTopLevel default value for the top level scope
+   * @param defaultForValidateFields default value for the top level scope
    * @return the compatibility level if found, otherwise null
    */
   default CompatibilityLevel compatibilityLevel(String subject,
                                                 boolean returnTopLevelIfNotFound,
-                                                CompatibilityLevel defaultForTopLevel)
+                                                CompatibilityLevel defaultForTopLevel,
+                                                boolean defaultForValidateFields)
       throws StoreException {
-    Config config = config(subject, returnTopLevelIfNotFound, new Config(defaultForTopLevel.name));
+    Config config = config(subject,
+        returnTopLevelIfNotFound,
+        new Config(defaultForTopLevel.name, defaultForValidateFields));
```

Comment:
Remove the changes in this file. The `defaultForValidateFields` is ignored.

Reply:
not sure if I follow. `defaultForValidateFields` is used in the constructor of `Config`. on a side note, this method seems to be unused. shall I remove this method altogether?